apps/test/unit/lib/levelbuilder/lesson-editor/SearchBoxTest.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import {mount, shallow} from 'enzyme';
+import sinon from 'sinon';
+import {expect} from '../../../../util/reconfiguredChai';
+import SearchBox from '@cdo/apps/lib/levelbuilder/lesson-editor/SearchBox';
+
+describe('SearchBox', () => {
+  let defaultProps, constructOptions, fetchSpy;
+
+  beforeEach(() => {
+    constructOptions = sinon.spy();
+    fetchSpy = sinon.stub(window, 'fetch');
+    defaultProps = {
+      onSearchSelect: () => {},
+      additionalQueryParams: {extraParam1: 1, extraParam2: 2},
+      searchUrl: 'fakesearch',
+      constructOptions
+    };
+  });
+
+  afterEach(() => {
+    fetchSpy.restore();
+  });
+
+  it('renders default props', () => {
+    const wrapper = shallow(<SearchBox {...defaultProps} />);
+    expect(wrapper.find('Async').length).to.equal(1);
+  });
+
+  it('searches when query is 3+ letters', () => {
```

Comment:
if you switch to `sinon.fakeServer`, then this could become an asynchronous test case, which has to call `done` in order to pass:
```suggestion
  it('searches when query is 3+ letters', done => {
```

Reply:
apparently calling done() indicates success, but you can also pass an error to it to indicate failure: https://github.com/code-dot-org/code-dot-org/blob/da7ec6b54e9c387f49c47afd8d826a7471e50578/apps/test/unit/lib/kits/maker/portScanningTest.js#L34